src/commands/ping.ts

```diff
@@ -0,0 +1,8 @@
+// Codey ping Command
+
+import Discord from 'discord.js';
+
+export const pingCmd = async (message: Discord.Message) => {
+  message.channel.send('pong');
+  return;
```

Comment:
you don't need return here if you are not returning anything

Reply:
the linter thing keeps giving me warnings (github actions) i dunno how to fix those

Reply:
it's because you need to specify a return type, so myabe like `async (message: Discord.Message): Promise<void> => ...`